test/config/tests.js

```diff
@@ -71,15 +71,22 @@ exports['eslint-plugin-zapier'] = {
     assert.doesNotThrow(() => linter.executeOnText(''));
   },
 
-  'exposes a prettier config': () => {
-    const linter = makeLinter('prettier');
+  'exposes a node config': () => {
+    const linter = makeLinter('node');
```

Comment:
could you add a couple of extra tests here that assert that the `node` config doesn't set the `browser` env and it does set the `node` env?